priv/index.html

```diff
@@ -5,43 +5,72 @@
 
 <title>BJukebox</title>
 
-<link href='https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,700,600' rel='stylesheet' type='text/css'>
+<link href='https://fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,700,600|Fascinate+Inline|Righteous' rel='stylesheet' type='text/css'>
```

Comment:
This is more of tin-foil hat paranoiac suggestion: can we serve the font file from ocf static instead of user's browser hitting the google server each time?

context: for privacy/just because, I personally setup the browsers to block outbound requests that's not a match to the url that they are visiting.

Reply:
Sure. It makes a lot of sense to go in that direction anyways since this is self-hosted and it's better for everything to be in one place. 

I'm curious as to what other people think about this before adding this as a commit. It seems like we can just serve [the fonts stylesheet](https://fonts.googleapis.com/css?family=Fascinate+Inline|Righteous|Source+Sans+Pro:400,600,700&display=swap) from a local file which link to the static fonts.

Reply:
If I were writing BJukebox today, I'd try to avoid loading external fonts. I try to go for minimal designs that the default OS fonts would work well in.

For a case like this, I don't have super strong opinions either way, but it's worth noting that extensions like [Decentraleyes](https://decentraleyes.org/) exist for those who want to reduce the tracking ability of CDNs.